src/components/WordVoting.vue

```diff
@@ -0,0 +1,559 @@
+              @click="showInitiate = !showInitiate"
+            >
+              <IconCheckmarkCircle />
+              <span class="desktop">{{ $t('views.WordDetail.Initiate') }}</span>
+              <span class="mobile">{{ $t('views.WordDetail.InitiateMobile') }}</span>
+            </AeButton>
+          </div>
+        </div>
+        <VoteCard
+          v-for="vote in votes"
+          :key="vote.id"
+          :vote="vote"
+          :data="data"
+          :contract-address="saleContractAddress"
+          :info="tokenInfo"
+        />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { mapState } from 'vuex';
+import BigNumber from 'bignumber.js';
+import { EventBus } from '../utils/eventBus';
+import Backend from '../utils/backend';
+import IconCheckmarkCircle from '../assets/iconCheckmarkCircle.svg?icon-component';
+import IconPlus from '../assets/iconPlus.svg?icon-component';
+import IconTokensBuySell from '../assets/iconTokensBuySell.svg?icon-component';
+import WordBuySellButtons from './WordBuySellButtons.vue';
+import AeButton from './AeButton.vue';
+import AeAmount from './AeAmount.vue';
+import FiatValue from './FiatValue.vue';
+import VoteCard from './VoteCard.vue';
+import MessageInput from './MessageInput.vue';
+import OutlinedButton from './OutlinedButton.vue';
+import TabBar from './TabBar.vue';
+import TabBarButton from './TabBarButton.vue';
+import ButtonPlain from './ButtonPlain.vue';
+import Loader from './Loader.vue';
+import { shiftDecimalPlaces, blockToDate } from '../utils';
+
+export default {
+  name: 'WordVoting',
+  components: {
+    AeButton,
+    AeAmount,
+    FiatValue,
+    IconCheckmarkCircle,
+    IconPlus,
+    IconTokensBuySell,
+    VoteCard,
+    MessageInput,
+    OutlinedButton,
+    TabBar,
+    TabBarButton,
+    ButtonPlain,
+    WordBuySellButtons,
+    Loader,
+  },
+  props: {
+    data: { type: Object, required: true },
+    selectedWord: { type: String, required: true },
+    saleContractAddress: { type: String, required: true },
+  },
+  data() {
+    return {
+      activeTab: 'ongoing',
+      description: '',
+      loading: false,
+      progressMessage: '',
+      newVotePayout: '',
+      ongoingVotes: [],
+      pastVotes: [],
+      myVotes: [],
+      showInitiate: false,
+    };
+  },
+  computed: {
+    ...mapState(['address', 'tokenInfo', 'tokenBalances']),
+    votes() {
+      switch (this.activeTab) {
+        case 'ongoing':
+          return this.ongoingVotes;
+        case 'past':
+          return this.pastVotes;
+        case 'my':
+          return this.myVotes;
+        default:
+          return [];
+      }
+    },
+    maxAmount() {
+      const tokenBalance = this.tokenBalances && this.data
+        && this.tokenBalances.find((b) => b.token === this.data.tokenAddress);
+
+      return tokenBalance ? tokenBalance.balance : '0';
+    },
+  },
+  watch: {
+    maxAmount() {
+      this.loadVotes();
+    },
+  },
+  async created() {
```

Comment:
```suggestion
  async mounted() {
```
to don't run it on server side in SSR mode